examples/ota-requestor-app/linux/main.cpp

```diff
@@ -100,6 +103,9 @@ OptionSet cmdLineOptions = {
     "  -c, --requestorCanConsent <true | false>\n"
     "       Value for the RequestorCanConsent field in the QueryImage command.\n"
     "       If not supplied, the value is determined by the driver.\n"
+    "  -d, --disableNotifyUpdateApplied\n"
+    "       If supplied, disable sending of the NotifyUpdateApplied command.\n"
+    "       Otherwise, after successfully loading into the updated image, send the NotifyUpdateApplied command.\n"
```

Comment:
```suggestion
    "       Otherwise, after successfully loading into the updated image, send the NotifyUpdateApplied command. For test scenarios only.\n"
```